src/Microsoft.AspNetCore.Razor.Language/src/Components/TypeNameHelper.cs

```diff
@@ -0,0 +1,59 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.AspNetCore.Razor.Language;
+internal class TypeNameHelper
+{
+    private static readonly string[] PredefinedTypeNames = new[] {"bool",
+        "int",
```

Comment:
```suggestion
    private static readonly string[] PredefinedTypeNames = new[] {
        "bool",
        "int",
```

Also, should we leave a comment about the fast pass logic below in case a new C# type is added (granted that's pretty unlikely).

Reply:
I don't think it's necessary given how unlikely it is, and wouldn't help discovery, since by the time something like that caused a bug, it will be self evident on the investigation